packages/plugin-ext/package.json

```diff
@@ -5,15 +5,16 @@
   "dependencies": {
     "@theia/core": "^0.3.8",
     "@theia/filesystem": "^0.3.8",
-    "@theia/workspace": "^0.3.8"
+    "@theia/workspace": "^0.3.8",
+    "ps-tree": "latest"
```

Comment:
Please use version that will be approved in eclipse CQ

Reply:
ok